### ISSUE TITLE
Hi,In your paper the generator loss function is huber loss but in your pytorch version, the loss function is L1 loss

### ISSUE THREAD

**Issue by meiguoofa** [richzhang/colorization-pytorch#19]
So the L1 loss function more suit for Colorization task?
